style.css

```diff
@@ -0,0 +1,141 @@
+#navbar{
+    background-color: rgb(145, 32, 32);
+    color: rgb(255, 255, 255);
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+
+
+}
+#navbar, ul{
+    list-style: none;
+  
+
+}
+/* #navbar, ul, li{
+   
+} */
+#navbar, ul, li, a{
+    color: rgb(255, 255, 255);
+    text-decoration: none;
+    padding: 2px 8px;
+    float: right;
+    text-align: center;
+    font-size: 16px;
+
+}
+
+
+ 
+
```

Comment:
please remove extra lines in your code and use only one line to separate your code blocks.